CHANGELOG.md

```diff
@@ -31,6 +31,36 @@
 
 ### Key Bug Fixes
 
+- Azure Communication Calling Service
+  - Fixed an issue where `handlePushNotification` did not return false if the same payload had been processed already.
+  - Improved logging to help identify the source of `hangup`-related issues reported in GitHub.
+  - Fixed an issue where the remote participant was still available after hangup/disconnect. [#134](https://github.com/Azure/Communication/issues/134)
+  - Removing `CommunicationUserCredentialPolicy`, this policy was a duplicate of cores `BearerTokenCredentialPolicy`.  
+  Communication now has new ability to create `BearerTokenCredentialPolicy` using the new `CommunicationPolicyTokenCredential`. 
+  - Communication identifier `MicrosoftTeamsUserIdentifier` property `identifier` renamed to `userId` since identifier was too generic.
+
+## 1.0.0-beta.7 (2021-01-12)
+
+### New Features
+
+- Azure Communication Calling Service
+  - Added the ability to set the Caller display name when initializing the library.
+
+- Azure Communication Common Library
+  - Added a new communication identifier `MicrosoftTeamsUserIdentifier`, used to represent a Microsoft Teams user.
+  - Introduced the new `CommunicationTokenRefreshOptions` type for specifying communication token refresh options.
+
+### Breaking Changes
+- Azure Communication Common Library
+  - Renamed the type `CommunicationUserCredential` to `CommunicationTokenCredential`, as it represents a token.
```

Comment:
Since these are technically going to be newly introduced to master I think these should move into the (upcoming) features sections of the change log.

Reply:
Moved, thanks!